tests/test_referenceObjectLoader.py

```diff
@@ -296,7 +296,7 @@ def setUpClass(cls):
 
         # override some field names.
         config = convertReferenceCatalogTestBase.makeConvertConfig(withRaDecErr=True, withMagErr=True,
-                                                                   withPm=True, withParallax=True)
+                                                                   withPm=True)
```

Comment:
This is a consequence of changing `ReferenceObjectLoader.makeMinimalSchema` so that `addProperMotion` adds the proper motion parameters **and** their errors. I removed the argument `withPmErr` from `convertReferenceCatalogTestBase.makeConvertConfig` because the proper motion errors are no longer added separately (so `withPm` adds both), so it is also removed here. I can include this in the commit message when I rebase and squash the messages.

Reply:
But this change removes `withParallax`? 😕

Reply:
`withParallax=True` is from one of the earlier commits on this PR. Including it was necessary because of the side-effects of how `ReferenceObjectLoader.makeMinimalSchema` worked before I refactored the ticket. Parallax is not actually tested in this class.